docs/compatibility.md

```diff
@@ -661,6 +661,10 @@ The GPU will use different precision than Java's toString method when converting
 types to strings. The GPU uses a lowercase `e` prefix for an exponent while Spark uses uppercase
 `E`. As a result the computed string can differ from the default behavior in Spark.
 
+The `format_number` function will retain 10 digits of precision for the GPU when the input is a floating 
```

Comment:
Can we have floating point be off by default then, so users can opt into it?

Reply:
done